examples/README.rst

```diff
@@ -0,0 +1,112 @@
+Bespoke linear operators
+=================================
+In this section we provide a range of pedagogical example applications of 
+``optimusprimal`` both on 1D signals (*e.g.* time series datasets) and 2D 
+signals (*e.g.* image processing datasets). In each case there are many 
+commonalities. The canonical structure of these applications is 
+
+**import/simulate data** :math:`\rightarrow` 
+**define a data-fidelity (likelihood)** :math:`\rightarrow` 
+**define a regulariser (prior)** :math:`\rightarrow` 
+**run the primal-dual solver**.
+
+Custom Measurement operators :math:`\Phi`
+-----------------------------------------
+For each application we adopt a standard de-noising setting, in which the 
+measurement (sensing) operator is simply the identity matrix. In practice 
+this operator is customized for a particular application. To do his one 
+simply needs to define a linear operator class which has the following 
+structure
+
+.. code-block:: python 
+
+    class my_custom_linear_operator:
```

Comment:
I think it might be better to include an example of a simple custom operator (that is actually code) instead of using a kind of template like this

Reply:
Added custom measurement operator, and a custom viewer examples.